### DISCUSSION TITLE
How to change the height of the content of a tab?

### DISCUSSION THREAD

**Discussion by xulihang** [NG-ZORRO/ng-zorro-antd#7666] [Q&A]
I would like to know how to change the height of the content of a tab to a value like "100vh".

I am trying with this but it does not work.

```html
<nz-tab nzTitle="Scan" class="scan-tab">
</nz-tab>
```

```css
.scan-tab {
  height: 100vh;
}
```

**Comment by catherinekennedy** [comment]
Hi,
In order to change the height of the content of a tab. you need to specify the `class="scan-tab"` in the `nz-tabset` and not in the `nz-tab`

![image](https://user-images.githubusercontent.com/67736813/235141809-ceb7190e-fe0a-4d6b-9960-f8061497a31b.png)
![image](https://user-images.githubusercontent.com/67736813/235141877-2d23554a-31dc-4d77-bf45-c31be61f0fa8.png)

I hope this will help you :)

**Reply by victourNwafor** [reply]
please how can I change the font-size of the nzTitle ?

**Reply by ParsaArvanehPA** [reply]
Hi @vicSoftcode 
The reason that op's style didn't change is due to Angular's [view encapsulation](https://angular.io/guide/view-encapsulation). Whenever you want to change an inner component's style, either one of Ng-zorro's components or even one of your own components from an uppper level, you need to use `::ng-deep`.

In this scenario, you need to write: 
```css
::ng-deep {
  .ant-tabs-tab-btn {
    font-size: 2.4rem;
  }
}
```

Also it is recommended to wrap your `::ng-deep` inside of `:host`, otherwise the scope of the `::ng-deep` will be global; but with `:host`, it only effects components styles' from the current node downward.
